### PR TITLE
New package Corel.MindManager v 23.1.243

### PR DESCRIPTION
**Pull request by FaserF**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138829)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-12T10:16:54.8831923Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0][NestedInstallerFiles] Description: Value type not permitted by 'type' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'NestedInstallerFiles'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: Corel.MindManager.installer.yaml Manifest Error: Failed to process field. [NestedInstallerFiles] Manifest Error: Required field missing. [NestedInstallerFiles] Manifest Error: Only one entry for NestedInstallerFiles can be specified for non-portable InstallerTypes. [NestedInstallerFiles]

(Automated response - build 720.)

**Review by Trenly — APPROVED** [review]

File: manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: MindManager 23.msi
+  InstallerUrl: https://download.mindjet.com/MindManager_64bit_23.1.243_Admin.zip
```
Comment (stephengillie) [inline manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml:19]:
```suggestion
  InstallerUrl: https://dwnld.mindmanager.com/stubs/Release/MindManager23/MindManager23_setup.exe
```

Hi @FaserF,

The `PackageUrl` gave me a very different `InstallerUrl`.

- Should this one be used instead?
- Can you show me where the ZIP installer was found?

Comment (FaserF) [reply manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml:19]: The MSI is the one recommended from the manufacturer for deployments. Therefore I choosed the MSI variant, which also is better for companys deploying the application with winget. 
https://download.mindjet.com/MindManager_Large_Scale_Deployment_Guide_v_15_0.pdf 

This is the always latest link for the MSI variant: https://www.mindmanager.com/mm-win-enterprise-admin-64

Nevertheless, I have now also added the exe parameters. Commit will follow shortly

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 731.)

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/138830#issuecomment-1939624199

**Comment by FaserF** [comment]
An error with the dependency is stopping the process early. Therefore not stale, package itself does work.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334960 2024-02-26 09:18:08.156 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330] 2024-02-26 09:18:26.114 [CLI ] Failed to remove installer file. Reason: remove: The process cannot access the file because it is being used by another process.: "C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Microsoft.EdgeWebView2Runtime.120.0.2210.121\5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e" 2024-02-26 09:18:26.402 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330] 2024-02-26 09:17:03.730 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:03.740 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:03.754 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:03.904 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:04.068 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:04.206 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:07.924 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330] 2024-02-26 09:17:10.302 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.318 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.330 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.344 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.359 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.376 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.539 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.553 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.566 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.728 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.741 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:10.902 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.065 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.230 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.243 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.258 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.272 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.287 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.445 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.607 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.662 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.702 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.853 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.884 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.904 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.952 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:11.977 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:12.017 [CLI ] Installer [X64,burn,Machine,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:12.175 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:12.206 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:12.333 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:12.347 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:18.119 [CLI ] Installer [X64,msi,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:18.119 [CLI ] Installer [X64,exe,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:20.305 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-26 09:17:20.306 [CLI ] No suitable installer found for manifest Microsoft.DotNet.Framework.DeveloperPack_4 with version 4.8.1 2024-02-26 09:17:20.591 [CLI ] Installer [X64,burn,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 09:17:20.592 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.33135.0

(Automated response - build 828.)

File: manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  NestedInstallerFiles:
+  - RelativeFilePath: MindManager 23.msi
+  InstallerUrl: https://download.mindjet.com/MindManager_32bit_23.1.243_Admin.zip
+  InstallerSha256: E4860ED16A89158D24FC8154262A9F84DB5E46CB20A5CDAEAE4888C8855F4196
+  ProductCode: '{3FB7C526-8F7A-4708-B753-CA866668FE2F}'
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://www1.mindjet.de/downloads/MM23/MindManager_32bit_23.1.243_redist.exe
+  InstallerSha256: 9806559585779F4F1B44837924B4010B653A9AA75C9023C7F41AD0D35D4B522D
+  ProductCode: '{3FB7C526-8F7A-4708-B753-CA866668FE2F}'
```
Comment (stephengillie) [inline manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml:45]:
```suggestion
Installers:
- Architecture: x64
  InstallerType: zip
  NestedInstallerType: msi
  NestedInstallerFiles:
  - RelativeFilePath: MindManager 23.msi
  InstallerUrl: https://download.mindjet.com/MindManager_64bit_23.1.243_Admin.zip
  InstallerSha256: 902192AA8FC6AB1E3E7495457E912B49F0A14ABAF26C3B0EF5A9F6EC53F825DD
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  ProductCode: '{9D0B3E25-1515-456E-839C-8516FAD6D1A8}'
- Architecture: x64
  InstallerType: exe
  InstallerUrl: https://www1.mindjet.de/downloads/MM23/MindManager_64bit_23.1.243_redist.exe
  InstallerSha256: DF53541B4B14F694D710139CCB9DC46800E052FA53AF4EA070D5654DDDA42028
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  ProductCode: '{9D0B3E25-1515-456E-839C-8516FAD6D1A8}'
  InstallModes:
  - silent
  InstallerSwitches:
    Silent: /S /v/qn
- Architecture: x86
  InstallerType: zip
  NestedInstallerType: msi
  NestedInstallerFiles:
  - RelativeFilePath: MindManager 23.msi
  InstallerUrl: https://download.mindjet.com/MindManager_32bit_23.1.243_Admin.zip
  InstallerSha256: E4860ED16A89158D24FC8154262A9F84DB5E46CB20A5CDAEAE4888C8855F4196
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
  ProductCode: '{3FB7C526-8F7A-4708-B753-CA866668FE2F}'
- Architecture: x86
  InstallerType: exe
  InstallerUrl: https://www1.mindjet.de/downloads/MM23/MindManager_32bit_23.1.243_redist.exe
  InstallerSha256: 9806559585779F4F1B44837924B4010B653A9AA75C9023C7F41AD0D35D4B522D
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
  ProductCode: '{3FB7C526-8F7A-4708-B753-CA866668FE2F}'
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
The errors state an architecture mismatch with the dependencies. Our `x86` validations are performed on 32-bit OSes, but both `Microsoft.DotNet.Framework.DeveloperPack_4` & `Microsoft.VCRedist.2015+.x64` appear to be 64-bit. 

I've suggested `Microsoft.VCRedist.2015+.x86` where applicable, but I'm not sure of a 32-bit alternative for `Microsoft.DotNet.Framework.DeveloperPack_4`. Even with this suggestion, the 32-bit packages will probably error again.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334960
> 2024-03-04 18:54:11.450 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330]
> 2024-03-04 18:54:32.232 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330]
> 2024-03-04 18:52:13.860 [CLI ] Package hash verification failed. SHA256 in manifest [5b01d964ced28c1ff850b4de05a71f386addd815a30c4a9ee210ef90619df58e] does not match download [510da7da566aa1ef74f591920af99262072f24575c564771883f65c60860c330]
> 2024-03-04 18:52:28.788 [CLI ] No suitable installer found for manifest Microsoft.DotNet.Framework.DeveloperPack_4 with version 4.8.1

(Automated response - build 861.)

**Pull request closed without merging**